spec/system/organization_system_spec.rb

```diff
@@ -41,10 +41,17 @@
         expect(page.find(".alert")).to have_content "Updated"
       end
 
-      it "cannot set a reminder day after deadline day" do
+      it "can set a reminder day after deadline day" do
         fill_in "organization_reminder_day", with: 12
         fill_in "organization_deadline_day", with: 1
         click_on "Save"
+        expect(page.find(".alert")).to have_content "Updated"
+      end
+
+      it "cannot set a reminder day on the deadline day" do
+        fill_in "organization_reminder_day", with: 15
+        fill_in "organization_deadline_day", with: 15
+        click_on "Save"
```

Comment:
Both of these don't seem to need to be system tests (which are big, slow and flaky). Once you've tested one success case and one error case, I don't think you need to add more business logic into these tests (that should be part of the model or a request test).

Reply:
Agreed.

Reply:
Fixed.